application/console/controllers/CronController.php

```diff
@@ -117,6 +117,13 @@ public function actionDeleteInactiveUsers()
         User::deleteInactiveUsers();
     }
 
+    public function actionSendInactiveUsersEmail()
+    {
+        /* @var $emailer Emailer */
```

Comment:
Unless I missed it, we probably need a way to bail out if `INACTIVE_USER_CONTACT_EMAIL` isn't defined. I'm assuming that at least one of the IDPs will not want this feature.

Reply:
Should `INACTIVE_USER_CONTACT_EMAIL` be used? I think there was some discussion to use `hr_notifications_email` or `notifier_email_to` instead?

Reply:
I think so. Seems the consensus was `HR_NOTIFICATIONS_EMAIL`. For the sake of clarity, it should be all-caps when specified as an environment variable. In Terraform, it is all lowercase. I was in the midst of Terraform work when the other conversation happened so that's why I used lowercase.

Let's keep it simple and not add the `bool` unless it's later warranted.